sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -76,17 +76,27 @@ class Breakpoint {
 		public SourceLocation Location { get; private set; }
 		public int LocalId { get; private set; }
 		public int RemoteId { get; set; }
-		public BreakPointState State { get; set; }
+		public BreakpointState State { get; set; }
+		public string StackId => $"dotnet:{LocalId}";
 
-		public Breakpoint (SourceLocation loc, int localId, BreakPointState state)
+		public static bool TryParseId (string stackId, out int id)
+		{
+			id = -1;
+			if (stackId?.StartsWith ("dotnet:") != true)
```

Comment:
should this use `InvariantCulture`?

Reply:
Ordinal really, I'll clean this up in the proxy